utils/rsaencryption/rsa_encryption_test.go

```diff
@@ -46,3 +53,49 @@ func TestPrivateKeyToByte(t *testing.T) {
 	require.NotNil(t, b)
 	require.Greater(t, len(b), 1024)
 }
+
+func TestConvertEncryptedPemToPrivateKey(t *testing.T) {
+	fileName := os.TempDir() + "/encrypted_private_key.json"
```

Comment:
What do you think of extracting the file name to a constant? If we don't and if we decide to change it, we can forget to change it somewhere

Reply:
@nkryuchkov  what do you mean by extract it?

Reply:
@guym-blox I mean we can create a constant with the file name, like the one below and use it.
```go
const EncryptedPrivateKeyFileName = "encrypted_private_key.json"
```

Reply:
@nkryuchkov I think it is used only in this test currently so this is not a must IMO.

Reply:
@y0sher also twice in `cli/generate_operator_keys.go`